### PR TITLE
Translation of schema and codelists

### PR DESCRIPTION
**Pull request by rhiaro**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -233,8 +236,32 @@ def copy_legacy_redirects(app, docname): # Sphinx expects two arguments
             if os.path.isfile(src_path):
                 copyfile(src_path, target_path)
 
-# -- Finally, Setup -------------------------------------------------------
+def translate_schema_and_codelists(language='en'):
```
Comment (odscjames) [inline docs/conf.py]: Should this function actually be called from somewhere, or does that come later?

File: requirements.in [hunk 2]
```diff
@@ -7,5 +7,6 @@ CommonMark==0.7.5 # Higher versions rename the package which mean we need code c
 -e git+https://github.com/OpenDataServices/sphinxcontrib-jsonschema.git@8d0d2bb00742b25707866b0f7a2b6c0c35948c5e#egg=sphinxcontrib-jsonschema
 -e git+https://github.com/OpenDataServices/sphinxcontrib-opendataservices.git@fab0ff0167d32ec243d42f272e0e50766299c078#egg=sphinxcontrib-opendataservices
 -e git+https://github.com/openownership/data-standard-sphinx-theme.git@038d4cef0a7baa63cc329d67093315ce4c931ca3#egg=standard_theme
+git+https://github.com/openownership/bods-babel.git@c391be94700cdaa50b915e23981f9058097d6bc3#egg=bods-babel
```
Comment (odscjames) [inline requirements.in]: Should this have a -e too?

**Comment by odscjames** [comment]
I added two comments but these are small things so if my comments aren't applicable then go for it

File: docs/conf.py (hunk 1 above)
Comment (rhiaro) [reply docs/conf.py]: That's coming soooooon!

**Pull request merged**